string name # Name of created mesh
---
bool success
